lib/pretty-repl.js

```diff
@@ -10,6 +11,10 @@ const ansiRegexMatchAll = ansiRegex();
 // Regex that matches ANSI escape sequences only at the beginning of a string.
 const ansiRegexMatchBeginningOnly = new RegExp(`^(${ansiRegexMatchAll.source})`);
 
+// Every open/close pair that should be matched against its counterpart for
+// highlighting.
+const allBrackets = '()[]{}\'"`$';
```

Comment:
The legacy shell did not highlight matching pairs of string quotes. I have no strong opinion here either.